### PR TITLE
sunshine: Add version 0.12.0

### PR DESCRIPTION
**Pull request by sitiom**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
64bit or 32bit? (in autoupdate)

File: bucket/sunshine.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "0.12.0",
+    "description": "A Gamestream host for Moonlight",
+    "homepage": "https://github.com/SunshineStream/Sunshine",
+    "license": "GPL-3.0-only",
+    "notes": [
+        "Run '$dir\\install-service.bat' to setup the service.",
+        "Sunshine can be configured at https://localhost:47990/ by default."
```
Comment (rashil2000) [inline bucket/sunshine.json:8]: Shouldn't this be http instead of https?

Comment (sitiom) [reply bucket/sunshine.json:8]: The instructions specifically say to use `https://xxx.xxx.xxx.xxx:47990` and ignore the "insecure website" website warning. I also tested it with `http` and it didn't work.

**Comment by sitiom** [comment]
> 64bit or 32bit? (in autoupdate)

Oops, fixed.

**Comment by rashil2000** [comment]
/verify

You should run /verify after making commits to run the automatic verifier

**Pull request merged**